repository.json

```diff
@@ -2119,6 +2119,19 @@
 					"tags": true
 				}
 			]
+		},
+		{
+			"name": "pycparser",
```

Comment:
The file is ordered by the `name` key to prevent merge conflicts (e.g. with #31). Please move your addition to the correct place.